### PR TITLE
chore: Add page for NINO

### PR DESCRIPTION
**Pull request by merlinc**
### What changed

- Add NINO page

### Issue tracking

- [OJ-1778](https://govukverify.atlassian.net/browse/OJ-1778)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[OJ-1778]: https://govukverify.atlassian.net/browse/OJ-1778?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/check/fields.js [hunk 1]
```diff
@@ -1 +1,19 @@
...
+      },
+      {
+        type: "maxlength",
+        fn: "maxlength",
+        arguments: [30],
+      },
+      // {
+      //   type: "alphaNumericWithSpecialChars",
+      //   fn: alphaNumericWithSpecialChars,
+      // },
```
Comment (scronk1) [inline src/app/check/fields.js:16]: This section necessary?

File: src/lib/config.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ module.exports = {
     PATHS: {
       SESSION: "session",
       AUTHORIZATION: "authorization",
+      CHECK: "/check",
```
Comment (scronk1) [inline src/lib/config.js:9]: Is the / in front of the check necessary? Doesn't seem to fit with the pattern of session and authorization above

File: tests/browser/features/happy.feature [hunk 3]
```diff
@@ -2,10 +2,11 @@ Feature: Happy path
 
   Successful journey through the system and back to the RP
 
-  @mock-api:success
+  @mock-api:success @wip
```
Comment (scronk1) [inline tests/browser/features/happy.feature:5]: Is @wip something we want to commit or was it for testing?

**Review by scronk1 — APPROVED** [review]

File: src/locales/cy/fields.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+nationalInsuranceNumber:
+  content: " "
+  label: What is your national insurance number?
```
Comment (merlinc) [inline src/locales/cy/fields.yml:3]: This is in English at the moment - translations are expected next week, and this isn't live yet, so it's safe to merge with the wrong language in it - for now.

File: src/app/check/fields.js (hunk 1 above)
Comment (merlinc) [reply src/app/check/fields.js]: Removed!

File: tests/browser/features/happy.feature (hunk 3 above)
Comment (merlinc) [reply tests/browser/features/happy.feature]: Removed!

File: src/lib/config.js (hunk 2 above)
Comment (merlinc) [reply src/lib/config.js]: Removed!

**Pull request merged**